pkg/engine/update.go

```diff
@@ -184,16 +186,34 @@ func installPlugins(
 	return allPlugins, defaultProviderVersions, nil
 }
 
-func installAndLoadPolicyPlugins(plugctx *plugin.Context, policies []RequiredPolicy) error {
+func installAndLoadPolicyPlugins(plugctx *plugin.Context, policies []RequiredPolicy, localPolicyPackPaths []string,
```

Comment:
For my own understanding, what's the difference between `RequiredPolicy` and `localPolicyPaths`? Is `RequiredPolicy` one provided from the service, which includes the policy pack's name and version? Whereas `localPolicyPackPaths` is just a slice of file paths, and we just load "whatever" happens to be located there?

Would it make sense to require the caller to just provide a slice of _one_ type of object? e.g. do we need the `[]string`? Or can we just require callers to convert the `[]string localPolicyPackPaths` to additional `RequiredPolicy` structs. (But just leaving "" for the name, version, etc.)

Ultimately it might be a way to make this function a little simpler to reason about, although with a name like `RequiredPolicy` perhaps it's better to leave the "local" packs as a separate param... since they are by definition optional?

Reply:
> Is `RequiredPolicy` one provided from the service, which includes the policy pack's name and version? Whereas `localPolicyPackPaths` is just a slice of file paths, and we just load "whatever" happens to be located there?

Right.

> Would it make sense to require the caller to just provide a slice of _one_ type of object? e.g. do we need the `[]string`? Or can we just require callers to convert the `[]string localPolicyPackPaths` to additional `RequiredPolicy` structs. (But just leaving "" for the name, version, etc.)

Ehh, maybe? I don't feel strongly about it either way. I slightly lean towards keeping `localPolicyPackPaths` as a separate option since it is a separate param and are optional.